src/discussions/learners/LearnerPostsView.jsx

```diff
@@ -88,7 +89,7 @@ function LearnerPostsView({ intl }) {
             <Spinner animation="border" variant="primary" size="lg" />
           </div>
         ) : (
-          nextPage && (
+          nextPage && loadingStatus === RequestStatus.SUCCESSFUL && (
```

Comment:
is this change related? Or does this fix some other issue?

Reply:
this is an unrelated change as seen from PR description. I think it saves us from making extra call when loading new thread is already in progress.